Package libncurses5-dev should be add be added to the toolchain linux script.

On Ubuntu 16.04.3 LTS when you run the toolchain script, the build fails with the following error:

```
gcc -shared -Wl,-soname,libhistory.so.7.0 -Wl,-rpath,/usr/local/lib -Wl,-soname,`basename libhistory.so.7.0 .0` -o libhistory.so.7.0 history.so histexpand.so histfile.so histsearch.so shell.so mbutil.so xmalloc.so xfree.so -lcurses
/usr/bin/ld: cannot find -lcurses
```

If you run the command:

```
sudo apt-get install libncurses5-dev
```

Then the build passes.

--- a/toolchains/install_linux.sh
+++ b/toolchains/install_linux.sh
@@ -33,4 +33,4 @@ check_cmds CMDS[@]
 
 exec_cmd "curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash"
 exec_cmd "nvm install v6.10.1"
-exec_cmd "sudo apt-get install -y autoconf automake build-essential curl chrpath git libreadline6-dev libtool zlib1g-dev libssl-dev libpq-dev nodejs python wget"
+exec_cmd "sudo apt-get install -y autoconf automake build-essential curl chrpath git libreadline6-dev libtool zlib1g-dev libssl-dev libpq-dev nodejs python wget libncurses5-dev"

